README.md

```diff
@@ -96,22 +96,31 @@ Also please note that this is not valid JSON. Just an example for how the output
  - Configure ParkAPI:
  
    Copy the [config_example.ini](config_example.ini) to `config.ini` and 
-   adjust the settings. 
-
- - Run the database migrations
+   adjust the settings as needed.
+   
+   The main categories in the config.ini are:
+   
+   - `DEFAULT`: These are the default values for all categories.
+   - `development`: Default when running the scraper or server.
+   - `testing`: Used when running the unittests.
+   - `production`: as well as any other key is activated through the 
+     *environment* of the operating system by setting a variable called `env`.
+   
+ - Run the database migrations once for development
         
-        $ yoyo apply --database postgresql://park_api:park_api@localhost/parkapi_testing schema/db/
+        (venv) $ PYTHONPATH=. python park_api/setupdb.py
+        
+   And once for testing
+   
+        (venv) $ PYTHONPATH=. env=testing python park_api/setupdb.py     
```

Comment:
Sorry for the nitpicking, this can be done similarly to running the server with
`(venv) $ env=... bin/parkapi-setupdb`. In that case the `PYTHONPATH` should not be required.

Reply:
I've just found it and am in the middle of rewriting. 

Still the PYTHONPATH=. is required. Maybe it's a python issue, i'm using 3.6.9 right now. Will check that, but commit the update in the meantime

Reply:
It's updated. And i tested python 3.8 which has the same problem. I think it's something that happened in python 3 in general. I always use `PYTHONPATH=.` for running scripts from the project root that are inside a sub-directory.

I would add a text like

In case of `ModuleNotFoundError: No module named 'park_api'` you need to set the
environment variable `PYTHONPATH` to the project root directory, e.g. `export PYTHONPATH=.` 

to the README but i'm actually not sure if that's the 'professional' way. 

One other thing is that the `psycopg2` is package is not longer supported for python 3.8 (an older version worked for 3.6.9). It's simple to fix by requiring `psycopg2-binary`. (Sorry for the mixture of topics, but it would fit into this pull request)

Reply:
I cannot reproduce these problems. The last [CI run](https://travis-ci.org/github/offenesdresden/ParkAPI/builds/773417576) (we have to migrate to GitHub Actions) shows that it's working with Python 3.7, 3.8 and 3.9. I just tested it and it works with Python 3.9.2 on my machine. Also psycopg2 is available without problems (although I don't see any problem switching to `psycopg2-binary`).

Reply:
If the problem persists I think you should open an issue for that so we can continue this conversation there.